receiver/varnishreceiver/scraper.go

```diff
@@ -59,6 +59,8 @@ func (v *varnishScraper) scrape(context.Context) (pdata.Metrics, error) {
 	now := pdata.NewTimestampFromTime(time.Now())
 	md := v.mb.NewMetricData()
 
+	md.ResourceMetrics().At(0).Resource().Attributes().UpsertString(metadata.A.CacheName, v.config.InstanceName)
```

Comment:
This is a direct index to index 0. Is that safe?

Reply:
The NewMetricData initializes the resource metric. If I use `md.ResourceMetrics().AppendEmpty().Resource().Attributes().UpsertString(metadata.A.CacheName, v.cacheName)` then I will have 2 resource metrics.